splitio/producer/initialization.go

```diff
@@ -281,6 +281,7 @@ func Start(logger logging.LoggerInterface, cfg *conf.Main) error {
 
 	cfgForAdmin := *cfg
 	cfgForAdmin.Apikey = logging.ObfuscateAPIKey(cfgForAdmin.Apikey)
+	cfgForAdmin.Storage.Redis.Pass = "xxxxxxxxxxxxxxx"
```

Comment:
is this line ok?

Reply:
Yes, it is.. we are hiding the Redis pass with those multiple "X"